drivers/ethernet/intel/CMakeLists.txt

```
@@ -0,0 +1,5 @@
+# Copyright (c) 2025 Intel Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library_sources_ifdef(CONFIG_ETH_INTEL_PLAT	eth_intel_plat.c)
+zephyr_library_sources_ifdef(CONFIG_ETH_INTEL_IGC	eth_intel_igc.c)
```

Mixing tabs and spaces? Also prefer to sort alphabetically

```suggestion
zephyr_library_sources_ifdef(CONFIG_ETH_INTEL_IGC	eth_intel_igc.c)
zephyr_library_sources_ifdef(CONFIG_ETH_INTEL_PLAT eth_intel_plat.c)
```

---

> Mixing tabs and spaces? Also prefer to sort alphabetically

Resolved in the new version. Please check.